content/reference/admin/private_locations/installation/gcp/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "GCP and Compute Engine"
+description: "How to install a Gatling Control Plane on GCP using Compute Engine, to set up your Private Locations and run load generators in your own GCP network"
+lead: "Run a Control Plane on GCP using Compute Engine, to set up your Private Locations and run load generators in your own GCP network"
+date: 2023-09-03T16:00:00+00:00
+lastmod: 2023-09-03T16:00:00+00:00
+weight: 22053
+---
+
+GCP [Compute Engine](https://cloud.google.com/compute/docs) is a computing and hosting service that lets you create and run virtual machines on Google infrastructure.
+
+In this example:
+- we use **Compute Engine** to configure a service to run the Gatling Control Plane
+- Compute Engine runs our Docker containers on a VM on the **GCP** infrastructure
+- the Control Plane configuration file is loaded from a **[Secret Manager](https://cloud.google.com/secret-manager)**
+
+This is only an example.
+
+{{< alert tip >}}
+We advise you to create a dedicated Gatling project, as permissions give access to secrets and existing Virtual Machines.
+{{< /alert >}}
+
+## Secret Manager
+
+{{< alert info >}}
+This section shows how to create a new secret on a [Secret Manager](https://cloud.google.com/secret-manager) for the control plane configuration, 
+skip if you already have a secret you want to use.
+{{< /alert >}}
+
+In the GCP management console, open the [Secret Manager](https://console.cloud.google.com/security/secret-manager) (or search for "Secret Manager" in the search bar). 
+Set the right project and click on "Create Secret".
+
+Choose a name for the secret, and upload the [Control Plane configuration]({{< ref "../../configuration/gcp" >}})
+
+Configure other options as preferred.
+
+Click Create secret.
+
+{{< img src="secret-configuration.png" alt="Configuring the secret" >}}
+
+## Service account
+
+We need a service account which will allow a VM to:
+
+- download the Control Plane's configuration file stored a secret
+- spawn new load generators on GCP when running a simulation
+
+### Role creation
+
+In the GCP management console, open [Roles](https://console.cloud.google.com/iam-admin/roles) (or search for "Roles" in the search bar).
+
+Required permissions: 
```

Comment:
Does the user need to enable these permissions? If so: "Enable the following required permissions:"